modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalNavTag.java

```diff
@@ -143,6 +153,9 @@ private void _renderVerticalNavItems(
 			if (active == null) {
 				active = Boolean.FALSE;
 			}
+			else if (_active == null) {
```

Comment:
this is a place where renaming `_active` to `_activeKey` pays off in terms of code clarity